### ISSUE TITLE
Public organizations are confusing

### ISSUE THREAD

**Issue by pierotofy** [DroneDB/Registry#259]
So the idea that organizations can be public is fine, but this concept needs more thoughts; it makes no sense to allow any user to create a public organization and allow that public organization to be seen in MY list of organizations (I've removed public orgs from the being listed).

We *can* have a separate page where we list the public organizations (for data discovery this can be interesting), but that needs to be separate from what users see in their space.

In short, this area needs improvement and further design considerations.
